src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -150,6 +151,7 @@ public static Task<RazorLanguageServer> CreateAsync(Stream input, Stream output,
                     .WithHandler<RazorProximityExpressionsEndpoint>()
                     .WithHandler<DocumentColorEndpoint>()
                     .WithHandler<FoldingRangeEndpoint>()
+                    .WithHandler<TextDocumentUriPresentationEndpoint>()
```

Comment:
Should we also be registering the text presentation endpoint here?

Reply:
![FacePalmOhNoGIF](https://user-images.githubusercontent.com/754264/167043560-cca94267-b9eb-4878-a77c-38483cc401a4.gif)